### PR TITLE
Modify org name for family integration stream

### PR DESCRIPTION
**Pull request by cshiwarale**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: team-config.yml [hunk 1]
```diff
@@ -689,11 +689,19 @@ et:
     build_notices_channel: "#et-builds"
   tags:
     application: employment-tribunals
-ds:
+fis-ds-web:
```
Comment (timja) [inline team-config.yml:692]: the key is the product, you should not have multiple apps under here.

use `fis`

```suggestion
fis:
```

File: team-config.yml [hunk 2]
```diff
@@ -689,11 +689,19 @@ et:
...
   team: "Family Integration Stream"
-  namespace: "ds"
+  namespace: "fis"
   slack:
     contact_channel: "#family-integration-devs"
     build_notices_channel: "#family-integration-cicd-notice"
   tags:
-    application: document-submission
+    application: fis-ds-web
+fis-bulk-scan-api:
```
Comment (timja) [inline team-config.yml:700]:
```suggestion
fis
```

**Review by timja — CHANGES_REQUESTED** [review]

**Comment by cshiwarale** [comment]
@timja I was referring to Adoption teams team-config as below, hence created multiple app

adoption-web:
  team: "Adoption"
  namespace:  "adoption"
  slack:
    contact_channel: "#adoption-tech"
    build_notices_channel: "#adoption-tech"
  tags:
    application: adoption-web
adoption-cos-api:
  team: "Adoption"
  namespace:  "adoption"
  slack:
    contact_channel: "#adoption-tech"
    build_notices_channel: "#adoption-tech"
  tags:
    application: adoption-cos-api
adoption-ccd-definitions:
  team: "Adoption"
  namespace:  "adoption"
  slack:
    contact_channel: "#adoption-tech"
    build_notices_channel: "#adoption-tech"
  tags:
    application: adoption-ccd-definitions

File: team-config.yml [hunk 3]
```diff
@@ -689,11 +689,19 @@ et:
...
   team: "Family Integration Stream"
-  namespace: "ds"
+  namespace: "fis"
   slack:
     contact_channel: "#family-integration-devs"
     build_notices_channel: "#family-integration-cicd-notice"
   tags:
-    application: document-submission
+    application: fis-ds-web
+fis:
```
Comment (timja) [inline team-config.yml:700]: Please remove duplicate

Comment (cshiwarale) [reply team-config.yml]: Removed duplicate

**Review by timja — APPROVED** [review]
You probably want to contact Bruce Rawstorne to confirm your application tag, that's likely not the right one but you can trivially update that after

**Pull request merged**